.devcontainer/devcontainer.json

```diff
@@ -5,5 +5,6 @@
 		"dockerfile": "Dockerfile"
 	},
 	"forwardPorts": [3000],
+	"postCreateCommand": "npm install",
```

Comment:
also add install for app/frontend directory

npm install; cd app/frontend; npm install